### PR TITLE
add PELE Network

### PR DESCRIPTION
**Pull request by jordanurbs**
Add space for PELE Network

Website: https://www.pele.network  
Telegram: https://t.me/PELENetwork
Twitter: https://twitter.com/PELENetwork

### CODE REVIEW

File: spaces/pele/index.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "token": "zil18f5rlhqz9vndw4w8p60d0n7vg3n9sqvta7n6t2",
```
Comment (bb111189) [inline spaces/pele/index.json]: This token belong to PORT token

**Review by bb111189 — CHANGES_REQUESTED** [review]

**Review by jordanurbs — COMMENTED** [review]
fix PELE token address

File: spaces/pele/index.json (hunk 1 above)
Comment (jordanurbs) [reply spaces/pele/index.json]: Fixed.

**Comment by jordanurbs** [comment]
Bump. Anything?

**Review by bb111189 — APPROVED** [review]

**Pull request merged**